QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+{
+	"@graph": [
+		{
+			"@id": "res:acqf",
+			"@type": "???:QualificationsFramework",
+			"ceasn:name": { "en": "African Continental Qualifications Framework" },
+			"ceterms:alternateName": { "en": "ACQF" },
+			"ceasn:description": { "en": "The ACQF is a comprehensive and inclusive meta-referencing qualifications framework, designed to support a holistic and systemic vision of learning, qualifications, and credentials. The ACQF is oriented to lifelong learning and supports parity of esteem of learning outcomes acquired in formal, non-formal and informal contexts, and all levels of education and training." },
+			"ceasn:dateValidFrom": "2022-11-01",
+			"ceterms:versionCode": "17",
+			"ceasn:source": [
+				"https://drive.google.com/file/d/1BnU5VOqS6hu2YRhowC7x6J1R1_p6UblV/view"
```

Comment:
Is this in our google drive?

Reply:
It should be fine for now.  It's just an internal document for discussion.